openfga/v1/openfga_service.proto

```diff
@@ -1420,6 +1420,18 @@ message ReadChangesRequest {
     (validate.rules).string.max_bytes = 5120,
     (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"eyJwayI6IkxBVEVTVF9OU0NPTkZJR19hdXRoMHN0b3JlIiwic2siOiIxem1qbXF3MWZLZExTcUoyN01MdTdqTjh0cWgifQ==\""}
   ];
+
+  google.protobuf.Timestamp start_time = 5 [
+    json_name = "start_time",
+    (validate.rules).timestamp.lt_now = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description:
+        "Start date and time of changes to read.\n"
+        "Format: ISO 8601 timestamp (e.g., 2022-01-01T00:00:00Z)\n"
+        "If a continuation token is provided, its value will be used in case it's timestamp is greater than the specified start time."
```

Comment:
> in case it's timestamp is greater than the specified start time

This is somewhat confusing and could cause conflicts once sort is exposed. We can simplify it by stating:
> If a continuation token is provided along side `start_time`, the continuation token will take precedence over `start_time`.`

Reply:
Yes, thanks for noticing. This was written before that change in requirements